src/api/app/views/layouts/webui/responsive_ux/_unlogged_options.html.haml

```diff
@@ -0,0 +1,52 @@
+- if can_signup
```

Comment:
Why do we have another sign up form in here? What's different to the partial we already have?

Reply:
Also you have a wild mixture of `sign_up` and `signup`. It's hard to keep track...

Reply:
We refactored this code. It is no longer repeated.